live-examples/css-examples/fonts/font-stretch.css

```diff
@@ -0,0 +1,12 @@
+@font-face {
+    src: url('/media/fonts/LeagueMono-VF.ttf') format('truetype');
+    font-family: 'League';
+    font-style: normal;
+    font-weight: 100;
```

Comment:
```suggestion
    font-weight: 400;
```

Reply:
No problem @wbamberg ! Ah yes, I must have have forgotten about changing the `font-weight` back haha, good catch. Changing it to `font-weight: 400` looks much better 👍